website/docusaurus.config.js

```diff
@@ -0,0 +1,132 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
```

Comment:
Should we keep those "help" comments (this one and the ones below)?

Reply:
Comments removed.